src/bun.js/ConsoleObject.zig

```diff
@@ -91,7 +91,7 @@ pub fn messageWithTypeAndLevel(
 ) callconv(JSC.conv) void {
     messageWithTypeAndLevel_(ctype, message_type, level, global, vals, len) catch |err| switch (err) {
         error.JSError => {},
-        error.OutOfMemory => global.throwOutOfMemory(),
+        error.OutOfMemory => global.throwOutOfMemory() catch {},
```

Comment:
can we add // TODO: properly propagate exception upwards.

i checked why this returns void and discovered that our main usage of this clears the exception immediately.